app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -73,9 +73,51 @@ If you are already a qualified teacher, you do not need to retrain to teach in E
 * to pass criminal and professional safeguarding checks (these will be organised by your employer)
 * a visa or immigration status allowing you to work in the UK
 
-### Your professional qualifications 
+## Get an international relocation payment (IRP) worth £10,000
+
+If you are interested in moving to the UK to teach languages or physics in an English secondary school, you could be eligible for a cash sum of £10,000 from the UK government to help with moving expenses, including visa costs and the [immigration health surcharge](https://www.gov.uk/healthcare-immigration-application). 
+
+The international relocation payment (IRP) will be available to non-UK citizens starting work in English schools in the academic year September 2023 to July 2024. You will not have to pay this money back and it can be spent in any way you wish. You can only receive the IRP once.
+
+### Check your eligibility for the IRP
```

Comment:
If all the following info is on the new fees and funding page, shouldn't we just be linking there rather than repeating it here? Couldn't we add a link along the lines of 'Find out more about the IRP' and include a jumplink to that section of the page?